src/JWT/JWT.cs

```diff
@@ -264,7 +264,12 @@ public static void Verify(string payloadJson, string decodedCrypto, string decod
             var secondsSinceEpoch = Math.Round((DateTime.UtcNow - UnixEpoch).TotalSeconds);
             if (secondsSinceEpoch >= expInt)
             {
-                throw new TokenExpiredException("Token has expired.");
+               var tokenExpiredException = new TokenExpiredException("Token has expired.")
```

Comment:
I'll try. I'll need to change my settings, cause Ctrl-K,D will result in tabs all over otherwise! 😛